documentation/modules/configuring/con-scaling-kafka-clusters.adoc

```diff
@@ -3,32 +3,44 @@
-For example, in a cluster of 12 brokers the pods are named `_cluster-name_-kafka-0` up to `_cluster-name_-kafka-11`.
-If you decide to scale down by one broker, the `_cluster-name_-kafka-11` will be removed.
-
-Before you remove a broker from a cluster, ensure that it is not assigned to any partitions.
-You should also decide which of the remaining brokers will be responsible for each of the partitions on the broker being decommissioned.
-Once the broker has no assigned partitions, you can scale the cluster down safely.
+Scaling Kafka clusters by adding brokers can increase the performance and reliability of the cluster. 
+Adding more brokers increases available resources, allowing the cluster to handle larger workloads and process more messages. 
+It can also improve fault tolerance by providing more replicas and backups. 
+Conversely, removing underutilized brokers can reduce resource consumption and improve efficiency. 
+Scaling must be done carefully to avoid disruption or data loss.
+By redistributing partitions across all brokers in the cluster, the resource utilization of each broker is reduced, which can increase the overall throughput of the cluster.
+
+NOTE: To increase the throughput of a Kafka topic, you can increase the number of partitions for that topic. 
+This allows the load of the topic to be shared between different brokers in the cluster. 
+However, if every broker is constrained by a specific resource (such as I/O), adding more partitions will not increase the throughput. 
+In this case, you need to add more brokers to the cluster.
+
+Change the `Kafka.spec.kafka.replicas` configuration to add or reduce the number of brokers.
+In the following example, the number of brokers is set at 3.
+A replica factor of 3 in Kafka means that each partition of a topic is replicated across three brokers. 
+This provides fault tolerance and ensures that if one of the brokers fails, other brokers are still available.
+
+.Example replica configuration
+[source,yaml,subs="+attributes"]
+----
+apiVersion: {KafkaApiVersion}
+kind: Kafka
+metadata:
+  name: my-cluster
+spec:
+  kafka:
+    replicas: 3
+ # ...
+----
+
+When you add add or remove brokers, Kafka does not automatically reassign partitions. 
+The best way to do this is using Cruise Control.
+You can use Cruise Control's `add-brokers` and `remove-brokers` modes when scaling a cluster up or down.
+
+* Use the `add-brokers` mode after scaling up a Kafka cluster to move partition replicas from existing brokers to the newly added brokers.
+* Use the `remove-brokers` mode before scaling down a Kafka cluster to move partition replicas off the brokers that are going to be removed.
+
+NOTE: When using a `StatefulSet` to manage Kafka brokers in a Kubernetes cluster, you cannot specify which specific pod to remove from the cluster. Instead, the broker removal process starts from the highest numbered pod in the `StatefulSet`.
```

Comment:
I guess that right now it's still valid with `StrimziPodSet` right @scholzj ? Should we specify it here as well?

Reply:
Yes. This is valid for the time being even with StrimziPodSets.

Reply:
Already raised by Paolo. This note is good, just remove the reference to StatefulSets.